ui/src/components/Orchestration/KeplrInitializer.tsx

```diff
@@ -0,0 +1,44 @@
+export const initializeKeplr = async () => {
+    await window.keplr.experimentalSuggestChain({
```

Comment:
how about passing in `window.keplr` explicitly rather than using ambient authority?

> modules should **not** export powerful objects (for example, objects that close over `fs` or `process.env`)
 -- [OCap Discipline](https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline)

`window` and `window.keplr` are in the same class as `fs` and `process` and `process.env`.

Reply:
another possibility is to export the data and call `window.keplr.experimentalSuggestChain` elsewhere.